github/orgs_projects.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ListProjectsOptions specifies the optional parameters to the
 // OrganizationsService.ListProjects method.
-type ListProjectsOptions struct {
+type ProjectsListOptions struct {
```

Comment:
Yes, the filename looks good. It's consistent with `repos_projects.go` and all the other `orgs_*.go` files.